2024-11-09 22:42:07.389 WARNING (Recorder) [homeassistant.components.recorder.util] The system could not validate that the sqlite3 database at //config/home-assistant_v2.db was shutdown cleanly
2024-11-09 22:42:07.534 WARNING (Recorder) [homeassistant.components.recorder.util] Ended unfinished session (id=664 from 2024-11-09 12:25:56.811385)
2024-11-09 22:42:20.076 WARNING (MainThread) [aioesphomeapi.reconnect_logic] Can't connect to ESPHome API for easy-skyfandc-2: Error resolving IP address: [Errno -3] Try again (APIConnectionError)
2024-11-09 22:42:36.191 WARNING (MainThread) [aioesphomeapi.reconnect_logic] Can't connect to ESPHome API for easy-skyfandc-1: Error connecting to [AddrInfo(family=<AddressFamily.AF_INET: 2>, type=<SocketKind.SOCK_STREAM: 1>, proto=6, sockaddr=IPv4Sockaddr(address='192.168.1.47', port=6053))]: [Errno 111] Connect call failed ('192.168.1.47', 6053) (SocketAPIError)
